sagemaker-debugger/debugger_interactive_analysis_profiling/interactive_analysis_profiling_data.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Can the output of this cell be removed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2174/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>

Reply:
<p>Why? This is the intended behavior of the <code>display_python_profile_stats</code>  function. The statistics for the top ten Python functions with the most cumulative time spent are displayed (perhaps we need to make that clearer here)</p><p> </p><p>We can shorten the output if it's too lengthy though. This would require updating the <code>display_python_profile_stats</code>  function defined earlier in the notebook.</p>

Reply:
I kept those outputs on purpose to show what kind of python profiling is done and what's operators are listed as a result. I made sure these are not including any security sensitive elements.

Reply:
I see, disregard my comment then. Typically it's cleaner to publish without cell outputs, but if you purposefully left it to illustrate the output, that is fine.